CONFIGURATION.md

```diff
@@ -101,10 +101,11 @@ The other placeholders are specified separately.
 [ preferred_ip_protocol: <string> | default = "ip6" ]
 
 # The query sent in the TCP probe and the expected associated response.
+# starttls upgrades TCP connection to TLS.
 query_response:
-  [ - [ [ expect: <string> ], [ send: <string> ] ], ... ]
+  [ - [ [ expect: <string> ], [ send: <string> ], [ starttls: yes ] ], ... ]
```

Comment:
This will need to be put on multiple lines now. The default is not yes, and this is a bool.